pkg/sfu/streamallocator/ratemonitor.go

```diff
@@ -16,6 +16,7 @@ package streamallocator
 
 import (
 	"fmt"
+	"sync"
```

Comment:
I should probably get rid of this completely and the time series as well I guess. Not really actively using it. 

But, really appreciate the clean up @paulwe .

Reply:
yeah it's only used in a debug log... i tried implementing an interface and using a noop version when debug logging is disabled but it crashed the go compiler. i took a bunch of samples from the sgp node that kept getting overloaded and this takes ~15% of cpu during spikes and i think the garbage from all the temp slices is causing issues. if we can get rid of it entirely that would be even better

Reply:
Wow, that is horrible. I will take a look and at least comment out its usage till I can find a better use for it. Thank you again.